src/utils.ts

```diff
@@ -105,3 +105,21 @@ function checkIsDevelopment() {
 export function log(message: any): void {
 	console.log(message);
 }
+
+const _keywords = [
```

Comment:
Btw, underscore prefixes for private are Dart, TypeScript uses private, I think this can be changed (I think both @devoncarew and I have both done this too... writing one language while thinking about another is confusing!).